lib/file-list.js

```diff
@@ -131,16 +131,16 @@ class FileList {
 
   // Check if we are currently refreshing
   _isRefreshing () {
-    return this._refreshing.isPending()
+    return this._refreshing && this._refreshing.isPending()
```

Comment:
`_isRefreshing` method is not used anywhere, we can remove it.

Reply:
I was working off HEAD and forgot about your changes to this file.

Reply:
No problem, just wanted to mention it :)